From 49a2d359da5016b90099cd2fa2e0f8b09896281a Mon Sep 17 00:00:00 2001
From: lucciolla200 <teoteo.odisharia@gmail.com>
Date: Tue, 28 Nov 2023 15:51:53 +0400
Subject: [PATCH] Changed 2 lines in secondFile.txt in git_2

---
 secondFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/secondFile.txt b/secondFile.txt
index 00935f1..79c1826 100644
--- a/secondFile.txt
+++ b/secondFile.txt
@@ -1,5 +1,5 @@
-Line 1
-Line 2
+update Line 1
+update Line 2
 Line 3
 Line 4
 Line 5
-- 
2.33.1.windows.1

